java/configuration/src/main/java/sleeper/configuration/properties/UserDefinedInstanceProperty.java

```diff
@@ -35,6 +35,7 @@ public enum UserDefinedInstanceProperty implements InstanceProperty {
     USER_JARS("sleeper.userjars"),
     TAGS_FILE("sleeper.tags.file"),
     TAGS("sleeper.tags"),
+    STACK_TAG_NAME("sleeper.stack.tag.name"),
```

Comment:
I think we should give this a default value of "DeploymentStack".

Reply:
Done